### PR TITLE
add hours in the transaction table date format

### PR DESCRIPTION
**Pull request by sangeetparashar**

### CODE REVIEW

File: src/app/components/connectButton.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  if (!isConnected) {
+    return (
+      <button
+        className="btn"
+        onClick={async () => {
+          // Disconnecting wallet first because sometimes when is connected but the user is not connected
+          if (isConnected) {
+            disconnect();
+          }
+          openConnectModal?.();
```
Comment (desamtralized) [inline src/app/components/connectButton.tsx]: why do we need an optional call here?

File: src/app/context/EthereumWalletContext.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    target() {
+      return {
+        id: "windowProvider",
+        name: "Window Provider",
+        provider: window.ethereum,
+      };
+    },
+  });
+  const { address, isConnected } = useAccount();
+  console.log("connector", connector);
```
Comment (desamtralized) [inline src/app/context/EthereumWalletContext.tsx]: can we please remove the log and the remaining commented lines?

File: src/app/context/EthereumWalletContext.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  //   const { connect } = useConnect({
+  //     connector: connector, // You can specify supported chains here
+  //   });
+  const { disconnect } = useDisconnect();
+
+  const connectEthWallet = () => {
```
Comment (desamtralized) [inline src/app/context/EthereumWalletContext.tsx]: any reason the body of this function is commented?

**Review by desamtralized — CHANGES_REQUESTED** [review]
looks good in general, added 2 comments

File: src/app/components/connectButton.tsx (hunk 1 above)
Comment (sangeetparashar) [reply src/app/components/connectButton.tsx]: removed

File: src/app/context/EthereumWalletContext.tsx (hunk 2 above)
Comment (sangeetparashar) [reply src/app/context/EthereumWalletContext.tsx]: removed

File: src/app/context/EthereumWalletContext.tsx (hunk 3 above)
Comment (sangeetparashar) [reply src/app/context/EthereumWalletContext.tsx]: removed

**Review by desamtralized — APPROVED** [review]

**Pull request merged**